app/assets/javascripts/discourse/app/templates/components/basic-topic-list.hbs

```diff
@@ -1,12 +1,4 @@
 {{#conditional-loading-spinner condition=loading}}
-  {{#if hasIncoming}}
```

Comment:
This is only used in `user-topics-list` so it shouldn't be here in the first place.